awx_collection/tests/integration/targets/tower_job_wait/tasks/main.yml

```diff
@@ -1,25 +1,107 @@
 ---
-- name: Launch a Job Template
+- name: generate random string for template and project
+  set_fact:
+    jt_name: "AWX-Collection-tests-tower_job_wait-long_running-{{ lookup('password', '/dev/null chars=ascii_letters length=16') }}"
+    proj_name: "AWX-Collection-tests-tower_job_wait-long_running-{{ lookup('password', '/dev/null chars=ascii_letters length=16') }}"
+
+- name: Assure that demo project exists
+  tower_project:
+    name: "{{ proj_name }}"
+    scm_type: 'git'
+    scm_url: 'https://github.com/ansible/test-playbooks.git'
+    scm_update_on_launch: true
+    organization: Default
+
+- name: Create a job template
+  tower_job_template:
+    name: "{{ jt_name }}"
+    playbook: "sleep.yml"
+    job_type: run
+    project: "{{ proj_name }}"
+    inventory: "Demo Inventory"
+
+- name: Check module fails with correct msg
+  tower_job_wait:
+    job_id: "99999999"
+  register: result
+  ignore_errors: true
+
+- assert:
+    that:
+      - result is failed
+      - "result.msg =='Unable to wait, on job 99999999 that ID does not exist in Tower.'"
+
+- name: Launch Demo Job Template (take happy path)
   tower_job_launch:
     job_template: "Demo Job Template"
   register: job
 
 - assert:
     that:
-      - "job is changed"
+      - job is changed
       - "job.status == 'pending'"
 
 - name: Wait for the Job to finish
   tower_job_wait:
     job_id: "{{ job.id }}"
-    timeout: 60
+  register: wait_results
 
-- name: Check module fails with correct msg
+# Make sure we worked and that we have some data in our results
+- assert:
+    that:
+      - wait_results is successful
+      - "'elapsed' in wait_results"
+      - "'id' in wait_results"
+
+- name: Launch a long running job
+  tower_job_launch:
+    job_template: "{{ jt_name }}"
+  register: job
+
+- assert:
+    that:
+      - job is changed
+      - "job.status == 'pending'"
```

Comment:
I dont know if we could get different statuses here (maybe 'running'?), but i feel like in the past we've had some flake due to asserting things like this, I like to avoid testing things like this unless they really provide value, like the canceled state below

Reply:
I'm good with removing these. They came over from the orig tests.